packages/notification-center/src/i18n/languages/fr.ts

```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const FR: ITranslationEntry = {
+  translations: {
+    notifications: 'Avis',
```

Comment:
In french we also just use 'Notification'
```suggestion
    notifications: 'Notifications',
```